python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -183,6 +191,17 @@ def __init__(
         # and this gets initialized in the connect method.
         self.hvac_state = None
 
+        # Keep track of whether a deviceEnabled log event needs to be sent
+        self.send_deviceEnabled_event = True
+
+        # Keep track of for which devices a config log event needs to be sent
+        self.send_config_event = set(
+            [hvac_topic.name for hvac_topic in list(HvacTopic)]
+        )
```

Comment:
You don't seem to be using this and I'm puzzled how you would use it.

If you want to know whether to send a config log event surely you can just iterate over all topic names and skip the ones that are not enabled? If you keep track of that info in a SAL event then it's always current and you are not duplicating info.

If you keep it please consider a name that sounds less like a boolean or a verb. I'd have to understand more what it's for to suggest anything. I'm not even sure that a "config log event" is.

Reply:
The idea was to keep track of for which devices the configuration has changed so a fooConfiguration event could be sent. Assuming that the behavior is the same as for the deviceEnabled event (i.e. it only gets sent if a field has changed) then I can get rid of this variable as well. So I'll remove it.